site/components/form/SearchSelect.tsx

```diff
@@ -32,6 +32,8 @@ interface SearchSelectProps<T> {
     width?: string;
     onFocus?: () => void;
     onBlur?: () => void;
+    closeMenuOnSelect?: boolean;
+    backspaceRemovesValue?: boolean;
```

Comment:
What's the expected behaviour for `backspaceRemovesValue`? 

I don't think this has added any different functionality to what was there before but I just want to double check in case I've misunderstood how it's meant to work.

Reply:
removed